modules/packages/DistributedBag.chpl

```diff
@@ -253,7 +254,20 @@ module DistributedBag {
       }
       return chpl_getPrivatizedCopy(unmanaged DistributedBagImpl(eltType), _pid);
     }
-
+  
+    // printing the contents of DistBag onto a channel
+    proc readWriteThis(ch: channel) throws {
+      ch <~> "[";
+      var iterations=0;
+      var size = this.getSize();
+      for i in this {
```

Comment:
You could alternatively do:

```suggestion
      for (i, iteration) in zip(this, 0..<this.getSize()) { {
```

Then you wouldn't have to create and update `iterations` in the loop